docs/01-get-started/loading-data-from-unified-graphql-schema.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Loading data from unified GraphQL schema
+description:
+  Explore the integration of a unified GraphQL schema in a Front-Commerce route
+  using Remix. This guide delves into the practical aspects of querying and
```

Comment:
**nitpick:** the sentence isn't very clear that it's about the app's unified schema exposed by FC

**suggestion:**

```suggestion
  Explore how to retrieve dynamic data from your application's unified GraphQL schema in a Front-Commerce route
  using Remix. This guide delves into the practical aspects of querying and
```